lf-to-abstract-sql.ometajs

```diff
@@ -307,6 +306,31 @@ export ometa LF2AbstractSQL <: SBVRCompilerLibs {
 		},
 	AttrNecessity :tableID =
 		Rule,
+	AttrDefinition :factType =
+			[	'Enum'
+				Value+:values
+			]
+
+			GetTable(factType):baseTable
+			(	?(baseTable === 'Attribute' || baseTable === 'ForeignKey')
+				GetTable(factType[0][1]):baseTable
+			)?
+			// Check we were able to find the base table, if not it's probably
+			// a definition on a term and not a fact type
+			?baseTable
+			FactTypeFieldName(factType):uniqueField
+			GetTableFieldID(baseTable, uniqueField):fieldID
+			?(fieldID !== false)
+			{baseTable.fields[fieldID].checks = baseTable.fields[fieldID].checks || []}:checks
+			{checks.push(
```

Comment:
Basically if it's an "enum" definition (eg `"a" or "b" or "c") then add a CHECK constraint like `field IN ('a', 'b', 'c')`